src/App.tsx

```diff
@@ -137,6 +139,12 @@ const App: FC = () => {
 
 export default () => (
   <AppSettingProvider>
-    <App />
+    {CLOUD ? (
```

Comment:
Before we restructure it like this - could you map out what the relationship between the User Account Provider and the app is?  Is this provider exclusively cloud-only or do parts of it need to be available in OSS?

For example, it looks to me like `AccountPage` contains the `UserAccountProvider` even though that's a shared route (at least according to SetOrg). I think we need to be 100% confident what is and isn't using the context before we move it conditionally.

Reply:
Upon checking and discussing with @wdoconnell , we have determined that this check is correct and wouldn't impact OSS negatively as the affected components/pages shouldn't be available to OSS without a copied/bookmarked link.

Reply:
This is a great solution to the problem of consistency between providers. I'm not worried about re-render impact of putting the provider above app since the `useEffect` hook there will only run when the default or current account changes (default account will rarely change, and when current account changes, the app is going to re-render anyway).  But we should watch for any unexpected rendering change until the account context is converted over to redux.